github/ipfs.yml

```diff
@@ -3443,9 +3436,8 @@ repositories:
       admin:
         - helia-dev
         - ipdx
-        - w3dt-stewards
-      pull:
-        - github-mgmt stewards
```

Comment:
@ipfs/ipdx any idea why these pull permissions got added everywhere in https://github.com/ipfs/github-mgmt/commit/fe64a02922a5bb3780e60177913dd62d9f521781?

Reply:
Yes! This is because `github-mgmt stewards` group is designated as moderator and security manager as per https://github.com/ipfs/github-mgmt/pull/189

So, unfortunately, they're going to come back, but you don't necessarily have to restore them yourself. The apply should go through anyway, and the config will be updated during the weekly sync.